bridges/snowbridge/pallets/system/src/lib.rs

```diff
@@ -663,6 +704,30 @@ pub mod pallet {
 			let secondary_exists = Channels::<T>::contains_key(SECONDARY_GOVERNANCE_CHANNEL);
 			primary_exists && secondary_exists
 		}
+
+		pub(crate) fn do_register_token(
+			asset_id: Location,
+			metadata: AssetRegistrarMetadata,
+			pays_fee: PaysFee<T>,
+		) -> Result<(), DispatchError> {
+			// Record the token id or fail if it has already been created
+			let token_id = TokenIdOf::convert_location(&asset_id)
+				.ok_or(Error::<T>::LocationConversionFailed)?;
+			Tokens::<T>::insert(token_id, asset_id.clone());
+			LocationToToken::<T>::insert(asset_id.clone(), token_id);
+
+			let command = Command::RegisterNativeToken {
+				token_id,
+				name: metadata.name,
+				symbol: metadata.symbol,
+				decimals: metadata.decimals,
+			};
+			Self::send(ParaId::from(1000).into(), command, pays_fee)?;
```

Comment:
Like the other permissionless governance apis, the message should be sent on the secondary governance channel.

```suggestion
			Self::send(SECONDARY_GOVERNANCE_CHANNEL, command, pays_fee)?;
```

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/128/commits/897fdadfe079f06f508dc39351162efacde86e46